src/pages/Synths/Overview/SynthChart/SynthChart.tsx

```diff
@@ -164,7 +163,10 @@ export const SynthChart: FC<SynthChartProps> = ({
 	);
 };
 
-const FrozenMessage = styled(GridDivCenteredRow)`
+const FrozenMessage = styled.div`
+	display: flex;
+	flex-direction: column;
+	align-items: center;
```

Comment:
Thanks for the fix, however I think its just missing `justify-items: center`.

The spacing between the icon and the text was taken from the design (meaning, the icon has more space from the text)